package.json

```diff
@@ -7,11 +7,19 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint . --fix"
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-navigation/native": "^5.9.3",
+    "@react-navigation/stack": "^5.14.3",
     "react": "16.13.1",
     "react-native": "0.63.4",
+    "react-native-gesture-handler": "^1.10.3",
+    "react-native-reanimated": "^2.1.0",
+    "react-native-safe-area-context": "^3.2.0",
+    "react-native-screens": "^3.0.0",
```

Comment:
I was actually wondering about whether this goes in dependencies or dev dependencies. What would be a good rubric on how to determine where it should belong for react-native apps? For web apps it's sort of like dependencies are .dll files that are necessary for runtime (they'll be bundled into the final bundle.js) vs the static .lib files necessary to build the bundle/executible files. In react native, isn't everything compiled down to a different language and binaries?

Reply:
That makes sense. I think about it in a similar way, but a little different. If any file is directly importing a specific package, it should go into the dependency array.
For example, `prettier` and `eslint` aren't actually ever imported in any of the `ts` or `tsx` files, so they should go into the devDependency array.

Obviously we aren't directly importing all of these libraries yet, but we will. We're actually currently importing the `gesture-handler` [here[(https://github.com/ihsan-project/ihsanReactNative/blob/main/index.js#L1) so that gives it away as well.

Another give-away is that the Expo CLI automatically adds these libraries to Expo React Native projects so devs don't need to think about it. Here's an example of what was autogenerated for one of my [sample Expo projects.](https://github.com/ahmednkhan24/React-Native-Bootcamp/blob/master/blog/package.json#L10)

Finally, if all else fails, `eslint` will actually throw a warning in the console when you run `yarn lint` and it finds a package in the devDependency array that should be in the dependency array.

Reply:
Thanks! I'm gonna take that and add it to the wiki somewhere